docs/get-started/try-viam/rover-resources/rover-tutorial/_index.md

```diff
@@ -0,0 +1,392 @@
+
+## What's inside the kit
+
+1. One assembled Viam Rover.
+
+   {{<imgproc src="get-started/try-viam/rover-resources/viam-rover-2/rover-side.png" resize="400x" declaredimensions=true alt="The side of the assembled Viam Rover">}}
+
+1. Four M2.5 screws for mounting your Raspberry Pi.
+
+   {{<imgproc src="get-started/try-viam/rover-resources/viam-rover/screws.jpg" resize="200x" declaredimensions=true alt="Four screws" >}}
+
+1. Two spare stiffer suspension springs.
+   You can swap them out with the springs that come with the rover if you need stiffer suspension for higher payload applications.
+
+   {{<imgproc src="get-started/try-viam/rover-resources/viam-rover/suspension-springs.jpg" resize="200x" declaredimensions=true alt="Two suspension springs" >}}
+
+1. Three different Allen wrenches (1.5 mm, 2 mm, and 2.5 mm) to unscrew the top and mount the Raspberry Pi.
+
+   {{<imgproc src="get-started/try-viam/rover-resources/viam-rover/allen-wrenches.png" resize="180x" alt="Three allen wrenches" >}}
+
+1. Four extenders to increase the height of the rover to house larger internal single board computers (such as a Jetson Orin Nano).
+
+   {{<imgproc src="get-started/try-viam/rover-resources/viam-rover-2/extenders.png" resize="400x" declaredimensions=true alt="Four extenders" >}}
+
+1. Ribbon cable for connecting the Raspberry Pi 4 to the Viam Rover 2 printed circuit board.
+
+   {{<imgproc src="get-started/try-viam/rover-resources/viam-rover-2/ribbon-cable.png" resize="400x" declaredimensions=true alt="Ribbon cable" >}}
+
+All together, your kit looks like this:
+
+{{<imgproc src="get-started/try-viam/rover-resources/viam-rover-2/box-contents.png" resize="400x" declaredimensions=true alt="A Viam Rover shipping box contents" >}}
+
+## Rover components
+
+### Dual drive motors with suspension and integrated motor encoders
+
+{{<imgproc src="get-started/try-viam/rover-resources/viam-rover/encoder-motors.jpg" resize="400x" declaredimensions=true alt="two motors with encoders" >}}
+
+The motors come with integrated encoders.
+For information on encoders, see [Encoder Component](/components/encoder/).
+For more information on encoded DC motors, see [Encoded Motors](/components/motor/gpio/encoded-motor/).
+
+The kit also includes stiffer suspension springs that you can substitute for the ones on the rover.
+Generally, a stiff suspension helps with precise steering control.
+In contrast, a soft suspension allows the wheels to move up and down to absorb small bumps on the rover's path.
+
+### Motor driver
+
+{{<imgproc src="get-started/try-viam/rover-resources/viam-rover/motor-driver.png" resize="400x" declaredimensions=true alt="A L298N motor driver" >}}
+
+The kit comes with an L298N driver dual H-Bridge DC motor driver.
+L298 is a high voltage and high current motor drive chip, and H-Bridge is typically used to control the rotating direction and speed of DC motors.
+
+### 720p webcam, with integrated microphone
```

Comment:
Was this comma intentional? I wasn't sure!

Reply:
Moved this over from the old one-- can change!